SIT.Manager.Avalonia/Views/SettingsPage.axaml.cs

```diff
@@ -10,5 +11,10 @@ public partial class SettingsPage : UserControl
             InitializeComponent();
             this.DataContext = App.Current.Services.GetService<SettingsPageViewModel>();
         }
+
+        private void ComboBox_SelectionChanged(object? sender, SelectionChangedEventArgs e)
+        {
+            UpdateTranslationStrings((Languages)languageComboBox.SelectedIndex);
+        }
```

Comment:
See previous comment about avoid events

Reply:
I just don't know how to use commands, this thing is complicated for me.